util/credits.txt

```diff
@@ -28,6 +29,7 @@ beardyunixer
 Beatriz Vital
 Ben Roberts
 ben-utzer
+Benjamin Lorteau
```

Comment:
This is me but I’d rather now see my real name for aesthetic reasons, can you please remove it @tobiasd?

Reply:
I'll regen the file and add an alias for you to the real you so you are only mentioned once.

Reply:
Thanks!